lib/std/build.zig

```diff
@@ -1775,6 +1775,22 @@ pub const LibExeObjStep = struct {
         }
     }
 
+    fn makePackageCmd(self: *LibExeObjStep, pkg: Pkg, zig_args: *ArrayList([]const u8)) anyerror!void {
```

Comment:
Is the `anyerror` necessary? As far as I can see, the only errors possible are `std.mem.Allocator.Error` a.k.a. `error.OutOfMemory`.